Some documentation tweaks (#1901)
tuist/tuist: website/markdown/apps-at-scale/2020-06-11-marek/post.mdx

### What code paradigms and architectures do you follow?

> We use MVVM architecture with [ReactiveSwift](https://github.com/ReactiveCocoa/ReactiveSwift) for all of our projects currently, although we have also started experimenting with [Composable architecture](https://github.com/pointfreeco/swift-composable-architecture) and Combine since we think that Redux-like approach might be a better option once we start using SwiftUI. Along with the MVVM architecture we have also started using [microfeatures](/docs/building-at-scale/microfeatures/) approach for one of our larger projects - this has had the benefit of better separation of concerns, faster build times, we have also introduced an example app for each individual feature which means it is extremely easy to concentrate on the task at hand. You can check out our [MVVM template](https://github.com/AckeeCZ/iOS-MVVM-ProjectTemplate/), hopefully we will be able to publish a revamped version with the microfeatures approach, and maybe composable architecture in the future.

Reviewer: ```suggestion
We use MVVM architecture with [ReactiveSwift](https://github.com/ReactiveCocoa/ReactiveSwift) for all of our projects currently, although we have also started experimenting with [Composable architecture](https://github.com/pointfreeco/swift-composable-architecture) and Combine since we think that Redux-like approach might be a better option once we start using SwiftUI. Along with the MVVM architecture we have also started using [microfeatures](/docs/building-at-scale/microfeatures/) approach for one of our larger projects - this has had the benefit of better separation of concerns and faster build times. We have also introduced an example app for each individual feature which means it is extremely easy to concentrate on the task at hand. You can check out our [MVVM template](https://github.com/AckeeCZ/iOS-MVVM-ProjectTemplate/), hopefully we will be able to publish a revamped version with the microfeatures approach, and maybe composable architecture in the future.
```

> We use MVVM architecture with [ReactiveSwift](https://github.com/ReactiveCocoa/ReactiveSwift) for all of our projects currently, although we have also started experimenting with [Composable architecture](https://github.com/pointfreeco/swift-composable-architecture) and Combine since we think that Redux-like approach might be a better option once we start using SwiftUI. Along with the MVVM architecture we have also started using [microfeatures](/docs/building-at-scale/microfeatures/) approach for one of our larger projects - this has had the benefit of better separation of concerns, faster build times, we have also introduced an example app for each individual feature which means it is extremely easy to concentrate on the task at hand. You can check out our [MVVM template](https://github.com/AckeeCZ/iOS-MVVM-ProjectTemplate/), hopefully we will be able to publish a revamped version with the microfeatures approach, and maybe composable architecture in the future.

Reviewer: ```suggestion
We use MVVM architecture with [ReactiveSwift](https://github.com/ReactiveCocoa/ReactiveSwift) for all of our projects currently, although we have also started experimenting with [Composable architecture](https://github.com/pointfreeco/swift-composable-architecture) and Combine since we think that Redux-like approach might be a better option once we start using SwiftUI. Along with the MVVM architecture we have also started using [microfeatures](/docs/building-at-scale/microfeatures/) approach for one of our larger projects - this has had the benefit of better separation of concerns, faster build times, we have also introduced an example app for each individual feature which means it is extremely easy to concentrate on the task at hand. You can check out our [MVVM template](https://github.com/AckeeCZ/iOS-MVVM-ProjectTemplate/): we are planning to publish a revamped version with the microfeatures approach and composable architecture in the future.
```
